### ISSUE TITLE
Долгое выполнение шага по проверке доступности для просмотра реквизита Комментарий

### ISSUE THREAD

**Issue by akademik1986** [Pr-Mex/vanessa-automation#2247]
Версия Vanessa Automation 1.2.041.36
Версия платформы 8.3.24.1758
Режим совместимости базы, где запускается Vanessa Automation 8.3.24
Операционная система windows

По какой то причине поле Комментарий долго проверяется шагами  (в конфигураторе Строка неограниченной длина, на всякий случай) 
И элемент "Комментарий" доступен не только для просмотра
И элемент "Комментарий" доступен только для просмотра
Вот данные ЖР
![image](https://github.com/user-attachments/assets/690188e0-d7fc-4b38-b2d3-81409ec7dd03)

Вот данные аллюра
![image](https://github.com/user-attachments/assets/655ab69d-f981-43ae-95ae-371ff2ae097d)

Вот фича
[Test ДОЛГО.zip](https://github.com/user-attachments/files/18146556/Test.zip)
, демонстрирующая долгое выполнение шага в фиче
А вот фича
[Test БЫСТРО.zip](https://github.com/user-attachments/files/18146562/Test.zip)
, где этот шаг закомментирован и выполнение происходит быстрое выполнение фичи

**Comment by Pr-Mex** [comment]
@akademik1986 
Попробуй искать элемент по имени, а не по заголовку.

**Comment by akademik1986** [comment]
@Pr-Mex да, по имени работает быстро! Обычно по имени стараюсь и использовать, но бывает плохо названы реквизиты, что легче тест понимать по имени, как в данном случае, но переделал комментарий по имени, спасибо. Не догадался попробовать в данном случае.
Закрываем?

**Comment by Pr-Mex** [comment]
@akademik1986 
Да, закрываем.
